### ISSUE TITLE
Development setup OS X - Update install steps - "No module named aes"

### ISSUE THREAD

**Issue by robertkarl** [spesmilo/electrum#1514]
It would be nice to improve the first-run development experience on Mac OS X. For example, where is the  `aes` dependency?

```
git clone git@github.com:robertkarl/electrum.git
cd electrum
sudo pip install dnspython
./electrum



Error: No module named aes. Try 'sudo pip install <module-name>'
```

I don't see the aes dependency listed here: https://electrum.org/#download

**Comment by robertkarl** [comment]
FWIW: previous ticket here https://github.com/spesmilo/electrum/issues/708

Also tried the following

    sudo pip install aes
    sudo pip install pyaes
    sudo pip install pythonaes
    sudo pip install aespython

And googling for the problem

**Comment by robertkarl** [comment]
Looks like aes is intended to refer to this python file: http://anh.cs.luc.edu/331/code/aes.py

**Comment by robertkarl** [comment]
OK, all that's needed is 
    python setup.py install

Unless I'm mistaken, these steps don't result in a working installation on OS X:

<img width="949" alt="screen shot 2015-10-24 at 6 21 32 am" src="https://cloud.githubusercontent.com/assets/1010817/10710638/9da4fa5e-7a17-11e5-9ec1-ec5750f14ecd.png">

**Comment by Thireus** [comment]
Maybe this will help: https://github.com/spesmilo/electrum/issues/1063

**Comment by robertkarl** [comment]
Nice writeup... would love to incorporate the steps needed into the public docs.

As far as just running `./electrum` from source, I think the only step I was missing was `python setup.py install`. Did you need the other steps listed in your post?

**Comment by Thireus** [comment]
To make some of the plugins work, yes.

**Comment by kyuupichan** [comment]
I can confirm the steps here worked perfectly for me on a fairly clean install of MacOSX Yosemite 10.10.5.

https://github.com/spesmilo/electrum/issues/1063#issuecomment-122604681

Thireus - any way you can get the install parts onto the main page or wiki?

**Comment by ecdsa** [comment]
the relevant module name is `slowaes`

**Comment by ecdsa** [comment]
we use pyaes now, so this issue should be solved: e1515743cdc52a068798b86ef4529def2b802ad8
